source/includes/wc-analytics/_reports.md

```diff
@@ -0,0 +1,731 @@
+# Analytics Reports #
+
+The reports API enables you to view analytics data on products, sales, revenue, orders, and more.
+
+## List all the reports endpoints ##
+
+This API helps you to list all the endpoints regarding reports.
+
+### HTTP request ###
+
+<div class="api-endpoint">
+	<div class="endpoint-data">
+		<i class="label label-get">GET</i>
+		<h6>/wp-json/wc-analytics/reports</h6>
+	</div>
+</div>
+
+> Example of how to list all the enpoints regarding WC Analytics reports:
+
+```shell
+curl https://example.com/wp-json/wc-analytics/reports \
+	-u consumer_key:consumer_secret
+```
+
+```javascript
+WooCommerce.get("reports")
+  .then((response) => {
+    console.log(response.data);
+  })
+  .catch((error) => {
+    console.log(error.response.data);
+  });
+```
+
+```php
+<?php
+print_r($woocommerce->get('reports'));
+?>
+```
+
+```python
+print(wcapi.get("reports").json())
+```
+
+```ruby
+woocommerce.post("reports").parsed_response
+```
+
+> JSON response example:
+
+```json
+[
+    {
+        "slug": "performance-indicators",
```

Comment:
should we document all properties as well for these reports?

Reply:
Thanks for the suggestion, I will add them in the upcoming PRs. Added as todo tasks it in the epic issue, along with the APIs for Import/Export as well: https://github.com/woocommerce/woocommerce-rest-api-docs/issues/248